### ISSUE TITLE
provider.aws: Not a valid region: eu-west-3 

### ISSUE THREAD

**Issue by pbramy** [terraform-providers/terraform-provider-aws#2735]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

Terraform v0.11.1
+ provider.aws v1.0.0
+ provider.null v1.0.0
+ provider.template v1.0.0

Downloading plugin for provider "aws" (1.6.0)...

### Affected Resource(s)
Please list the resources as a list, for example:
- All resources affected 

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```
erraform {
  backend "s3" {
    bucket = "deploystate"
    key    = "AWS/PARIS/tfstate"
    region = "eu-west-3"
  }
}
.../...

### Expected Behavior
What should have happened?
a new AWS region has been open in PARIS
( http://docs.aws.amazon.com/general/latest/gr/rande.html )

EU (Paris) | eu-west-3 | autoscaling.eu-west-3.amazonaws.com | HTTP and HTTPS

### Actual Behavior
What actually happened?
Cannot refer to eu-west-3 region

provider.aws: Not a valid region: eu-west-3 
or
Error configuring the backend "s3": Not a valid region: eu-west-3

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by VEBERArnaud** [comment]
related to #2707

**Comment by pbramy** [comment]
Arnaud, Thank you a lot for your quick answer. Do you know when it could be available ? How can we help ? please let me know. Merci d'avance!

**Comment by jen20** [comment]
Fixed by #2707 (I found the ELB ID by creating one and looking at the resulting policies).

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.7.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#170-january-12-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
